RCS file: /home/wine/wine/dlls/ole32/stubmanager.c,v
retrieving revision 1.19
diff -u -r1.19 stubmanager.c
--- ./dlls/ole32/stubmanager.c	17 Mar 2005 10:26:20 -0000	1.19
+++ ./dlls/ole32/stubmanager.c	10 Jun 2005 07:45:57 -0000
@@ -477,7 +477,7 @@
     default:
         WARN("object OID %s already unmarshaled\n",
             wine_dbgstr_longlong(m->oid));
-        ret = FALSE;
+        ret = TRUE;
         break;
     }
